src/getHumanAge.test.js

```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array ', () => {
+    const goals = getHumanAge(0, 0);
+
+    expect(goals).toBeInstanceOf(Array);
+  }); /* normaly this test is not necessary because of next test */
+
+  test('should return an expected HumanAge', () => {
+    const goals = getHumanAge(15, 15);
+
+    expect(goals).toEqual([1, 1]);
+  });
+
+  test('shoul return [0, 0], if cat&dog have 0 <= year < 1', () => {
```

Comment:
```suggestion
  test('should return [0, 0], if cat&dog have 0 <= year < 1', () => {
```

Reply:
I made a mistake in the title, I wrote " Solutin". sorry for that.